ros2doctor/ros2doctor/api/__init__.py

```diff
@@ -26,15 +27,93 @@
 from ros2doctor.api.format import doctor_warn
 
 
+# List of Environment Variables compiled from github.com/ros2/ros2cli/issues/1046
+# TODO(@fujitatomoya): In the future maybe get from centralized storage via rcl/rmw interfaces.
+RCL_ENVIRONMENT_VARIABLES: Final = [
+    'RMW_IMPLEMENTATION',
+    'ROS_AUTOMATIC_DISCOVERY_RANGE',
+    'ROS_STATIC_PEERS',
+    'ROS_DISABLE_LOANED_MESSAGES',
+    'ROS_DOMAIN_ID',
+    'ROS_SECURITY_ENABLE',
+    'ROS_SECURITY_STRATEGY',
+    'ROS_SECURITY_KEYSTORE',
+    'ROS_SECURITY_ENCLAVE_OVERRIDE',
+    'ROS_LOG_DIR',
+    'ROS_HOME',
+    'RCL_LOGGING_SPDLOG_EXPERIMENTAL_OLD_FLUSHING_BEHAVIOR'
+]
```

Comment:
We have 2 env vars for tracing:

1. `ROS_TRACE_DIR`: https://github.com/ros2/ros2_tracing#tracing
2. `TRACETOOLS_RUNTIME_DISABLE`: https://github.com/ros2/ros2_tracing#disabling-the-tracer-at-runtime